components/ma-edit-new.vue

```diff
@@ -725,6 +725,13 @@
                                         <textarea
                                             name="collection_method_text"
                                             placeholder=""
+                                            @change="
+                                            onZoneFieldChanged(
+                                                'description',
+                                                index,
+                                                $event.target.value,
+                                            )
+                                        "
```

Comment:
@catalyzecomms it seems you implemented @change only on the text area but I think you should implement it on all three fields